moveit_setup_assistant/include/moveit/setup_assistant/tools/moveit_config_data.h

```diff
@@ -327,6 +327,12 @@ class MoveItConfigData
   bool outputROSControllersYAML(const std::string& file_path);
   bool output3DSensorPluginYAML(const std::string& file_path);
 
+  /**
+   * \brief Helper function to get the controller that is controlling the joint
+   * \return controller type
+   */
+  std::string getJointHardwareInterface(std::string joint_name);
```

Comment:
strings should always be passed in with const &

Reply:
mb :sweat_smile: